source/parameters/floorshift.md

```diff
@@ -0,0 +1,16 @@
+(s:floorshift.params)=
+## FloorShiftP:  FloorShift Parameters
+
+The `FloorShiftP` parameter group describes the shift the reference curve of a `FloorShift` element.
+The components of `FloorShiftP` are:
+```{code} yaml
+  x_offset
+  y_offset
+  z_offset
+  t_offset
+  
+  x_rot
+  y_rot
+  z_rot
+```
```

Comment:
I think it will be so much easier for people to implement this if we take the time to write complete element examples, e.g.,
```suggestion
```{code} yaml
- floor_shift_1:
    kind: FloorShiftP
    x_offset: 0.0
    y_offset: 1.0
    z_offset: 0.0
    t_offset: 0.1
    x_rot: 0.0
    y_rot: 0.0
    z_rot: 0.0
```
```

And we should also note which default values are used if a key is not specified. Here for example, they can all be defaulted to zero if omitted.

Reply:
And we should also note which default values are used if a key is not specified. Here for example, they can all be defaulted to zero if omitted.

Reply:
Yes, let's check in in an hour :+1: